archunit/src/test/java/com/tngtech/archunit/lang/syntax/elements/MembersShouldTest.java

```diff
@@ -342,7 +342,7 @@ public void property_predicates(MembersShouldConjunction<?> conjunction, Set<Str
                 .evaluate(importClasses(ClassWithVariousMembers.class, A.class, B.class, C.class, MetaAnnotation.class));
 
         Set<String> actualMembers = parseMembers(result.getFailureReport().getDetails());
-        assertThat(actualMembers).containsOnlyElementsOf(expectedMembers);
+        assertThat(actualMembers).hasSameElementsAs(expectedMembers);
```

Comment:
It says "**like** obsolete explicit...", doesn't say exclusively :wink: The change also comes from Java 8 migration, because that enabled AssertJ upgrade which in turn made the old method here deprecated :wink: I thought about splitting it into a separate commit, but then thought it's just two little changes and doesn't need much explanation and comes from the same reason :man_shrugging:

Reply:
I believe that an earlier version of this commit did in fact remove quite some _obsolete explicit type arguments that can now be inferred_ in addition to this deprecation fix, but after rebasing behind 0a90c1d7d29468ee5f5e38787e0ecbfe4c274597, only the two deprecation fixes are left, for which I thought the commit message did not match.